generated/proto/namespace/namespace.proto

```diff
@@ -18,6 +18,8 @@ message NamespaceOptions {
 	bool needsRepair         = 5;
 
 	RetentionOptions retentionOptions = 6;
```

Comment:
could we rename `needsFilesetCleanup` too? did a [quick test](https://github.com/prateek/proto_rename_test) to ensure field rename is wire compatible.

Reply:
actually, can we rename em all to follow the `...Enabled` convention?

Reply:
done